front/app/containers/Admin/settings/customize/Header/HeaderImageDropzone.tsx

```diff
@@ -72,10 +72,13 @@ const HeaderImageDropzone = ({
     return ratio;
   };
 
+  const theme: any = useTheme();
   const overlayColor =
-    latestAppConfigStyleSettings?.signedOutHeaderOverlayColor;
+    latestAppConfigStyleSettings?.signedOutHeaderOverlayColor ||
+    theme.colorMain;
   const overlayOpacity =
-    latestAppConfigStyleSettings?.signedOutHeaderOverlayOpacity;
+    latestAppConfigStyleSettings?.signedOutHeaderOverlayOpacity ||
```

Comment:
This is problematic if the opacity is 0, it would coerce to false and go to the theme default value.

We need `typeof latestAppConfigStyleSettings?.signedOutHeaderOverlayOpacity === 'number' ? latestAppConfigStyleSettings.signedOutHeaderOverlayOpacity : theme...`

Reply:
@brentguf  Yes, completely forgot about it, thanks.

Won't `??` do the trick?

Reply:
How would you write that?

Reply:
Yes, I prefer to be more explicit because that reduces the changes of introducing bugs because of type coercion and/or misunderstanding of the code. 

?? is not used often and easy to misinterpret, but should work here.